addOns/grpc/grpc.gradle.kts

```diff
@@ -6,6 +6,18 @@ zapAddOn {
     manifest {
         author.set("ZAP Dev Team")
         url.set("https://www.zaproxy.org/docs/desktop/addons/grpc-support/")
+        extensions {
+            register("org.zaproxy.zap.extension.grpc.internal.ExtensionGrpcWebSocket") {
```

Comment:
It needs to be updated accordingly.

Reply:
i made all the changes , but still didn't work

Reply:
Push them.